docs/specification/draft/basic/transports.mdx

```diff
@@ -92,7 +92,7 @@ MCP endpoint.
 2. The client **MUST** include an `Accept` header, listing both `application/json` and
    `text/event-stream` as supported content types.
 3. The body of the POST request **MUST** be a single JSON-RPC _request_, _notification_, or _response_.
-4. If the input is a JSON-RPC _response_ or _notification_:
+4. If the input is a JSON-RPC _response_ for an eariler request, or it is a _notification_:
```

Comment:
Typo: "eariler" => "earlier".  However, I think it might be better change the line above this one instead.  See my next comment.